test/testunusedvar.cpp

```diff
@@ -1962,6 +1965,54 @@ class TestUnusedVar : public TestFixture {
                       errout_str());
     }
 
+    void structmember26() { // #10305
+        checkStructMemberUsage("struct S {\n"
+                               "    const int* p{};\n"
+                               "};\n"
+                               "struct C {\n"
```

Comment:
I don't understand the point to have struct C. a function is less code.

Reply:
I think I tried to imititate the code in the ticket, but I agree it's unnecessary.